src/lib.rs

```diff
@@ -338,13 +338,12 @@ pub fn start_bottom(enable_error_hook: &mut bool) -> anyhow::Result<()> {
     *enable_error_hook = true;
 
     let mut stdout_val = stdout();
-    execute!(
-        stdout_val,
-        Hide,
-        EnterAlternateScreen,
-        EnableBracketedPaste,
-        EnableMouseCapture,
-    )?;
+    execute!(stdout_val, Hide, EnterAlternateScreen, EnableBracketedPaste)?;
+    if app.app_config_fields.disable_click {
+        execute!(stdout_val, DisableMouseCapture)?;
```

Comment:
Wouldn't it be enough to just skip `EnableMouseCapture` if `disable_click` is set?

Also not sure if we need to skip `DisableMouseCapture` if `disable_click` is set on cleanup.

Reply:
With regards to the former, it's likely worth being explicit. With regards to the former, I can adapt this to run conditionally instead.

Reply:
Sounds reasonable, thanks!

Reply:
Done. I've left the unconditional `DisableMouseCapture` in place for `reset_stdout()` since it seems the sensible action in case of a panic.